zion:
  stair_modeling:
    ros__parameters:

      voxel_filter:
        leaf_size_xy: 0.04    
        leaf_size_z: 0.08     

      crop_box:
        min_x: 0.0
        max_x: 5.0
        min_y: -0.4
        max_y: 0.4
        min_z: -3.0
        max_z: 1.0

      topic_names:
        input_point_cloud_topic: "/zedm/zed_node/point_cloud/cloud_registered"

      frame_ids:
        input_cloud_frame: "zedm_left_camera_frame"
        output_cloud_frame: "zedm_base_link_projected"

      segmentation:
        distance_threshold: 0.05
        max_iterations: 500
        angle_threshold: 5.

      clustering:
        cluster_tolerance: 0.08
        min_cluster_size: 100
        max_cluster_size: 25000

      floor_finding:
        k_neighbors: 50

      avg_x_calculation:
        x_neighbors: 25
        y_threshold: 0.05

/**:
    ros__parameters:
      allNodeCanSee: True